fluentd/Dockerfile

```diff
@@ -69,7 +72,9 @@ RUN if [ ! -d /etc/fluent/plugin ] ; then \
         cp $sniffer /etc/fluent/plugin/ ; \
     fi
 
-COPY utils/** ${HOME}/utils
+ADD patch/logger/log.rb ${GEM_HOME}/gems/fluentd-0.12.43/lib/fluent
+ADD patch/logger/supervisor.rb ${GEM_HOME}/gems/fluentd-0.12.43/lib/fluent
```

Comment:
maybe add the `/` at the end?  `ADD patch/logger/supervisor.rb ${GEM_HOME}/gems/fluentd-0.12.43/lib/fluent/`
every other place that uses `ADD` for a directory has the trailing `/`